tests/perf-test/OptionsListUtils.perf-test.ts

```diff
@@ -75,4 +82,148 @@ describe('OptionsListUtils', () => {
         await waitForBatchedUpdates();
         await measureFunction(() => OptionsListUtils.getMemberInviteOptions(personalDetails, mockedBetas, SEARCH_VALUE));
     });
-});
+
+
+    /* formatSectionsFromSearchTerm performance tests */
```

Comment:
I think this is not necessary as its in the description. Also only one blank space is enough
```suggestion
```